examples/reporters/CMakeLists.txt

```diff
@@ -0,0 +1,127 @@
+add_executable(reporters_cpp_env
```

Comment:
I apologise for all the boilerplate this requires! 😛 One day we could script/automate this (although it's not worth the time right now)

Reply:
At least this sort of thing really only needs to be written out once unless you make a lot of structural changes!